runtime/src/main/java/org/corfudb/protocols/wireprotocol/ICorfuPayload.java

```diff
@@ -93,7 +93,7 @@
                             return new StreamAddressSpace(trimMark, map);
                         } catch (IOException ioe) {
                             throw new SerializerException("Exception when attempting to " +
-                                    "deserialize stream address space.");
+                                    "deserialize stream address space: " + ioe.getMessage());
```

Comment:
Just wrap the ieo with SerializerException, otherwise you lose the stack trace. 

`throw new SerializerException(ioe)`

Reply:
Probably need to overload 

```
public SerializerException(Throwable t) {
        super(t);
    }
```

Reply:
Sure, it's cleaner. Done.